core/src/apps/nostr/sign_event.py

```diff
@@ -0,0 +1,54 @@
+from typing import TYPE_CHECKING
+
+from apps.common.keychain import auto_keychain
+
+if TYPE_CHECKING:
+    from trezor.messages import NostrEventSignature, NostrSignEvent
+
+    from apps.common.keychain import Keychain
+
+
+@auto_keychain(__name__)
+async def sign_event(msg: NostrSignEvent, keychain: Keychain) -> NostrEventSignature:
+    from ubinascii import hexlify
+
+    from trezor import TR
+    from trezor.crypto.curve import secp256k1
+    from trezor.crypto.hashlib import sha256
+    from trezor.messages import NostrEventSignature
+    from trezor.ui.layouts import confirm_value
+
+    from apps.common import paths
+
+    address_n = msg.address_n
+    created_at = msg.created_at
+    kind = msg.kind
+    tags = [[t.key] + ([t.value] if t.value else []) + t.extra for t in msg.tags]
+    content = msg.content
+
+    await paths.validate_path(keychain, address_n)
+
+    node = keychain.derive(address_n)
+    pk = node.public_key()[-32:]
+    sk = node.private_key()
+
+    title = TR.nostr__event_kind_template.format(kind)
+
+    # confirm_value on TR only accepts one single info item
```

Comment:
would `confirm_properties` be a better choice?

Reply:
`confirm_properties` is nice but I like having it on 2 pages - one page for the actual content, and an extra page if you want to see the timestamp and tags...

The solution really is to fix `confirm_value` to accept an arbitrary number of properties in the "info" screen.